internal/lookout/ui/src/containers/lookoutV2/JobsTableContainer.tsx

```diff
@@ -234,7 +234,7 @@ export const JobsTableContainer = ({
   // Retrieve data for any expanded rows from intial query param state
   useEffect(() => {
     const rowsToFetch: PendingData[] = [
-      { parentRowId: "ROOT", skip: 0 },
+      { parentRowId: "ROOT", skip: pagination.pageIndex * pagination.pageSize },
```

Comment:
Separately, this makes Lookout load the correct set of rows when navigating to a URL containing a nonzero page index.